.storybook/stories/tree/tree.stories.ts

```diff
@@ -14,7 +14,7 @@ import { setupStorybook } from '../../helpers/setup-storybook.helpers';
 const defaultStory: Story = args => ({
   template: `
     <clr-tree>
-      ${getFileTreeNodeMarkup(filesRoot)}
+      ${getFileTreeNodeMarkup(filesRoot, { asLink: args.asLink })}
```

Comment:
```suggestion
      ${getFileTreeNodeMarkup(filesRoot, args)}
```

Why not just pass the whole `args` object? (Same thing the tree node story.)

Reply:
corrected in commit e22dd2e